crypto/constants.ts

```diff
@@ -14,6 +14,255 @@ let prefixes = {
   "zkappUri": "MinaZkappUri********",
   "deriveTokenId": "MinaDeriveTokenId***"
 };
+let prefixHashes = {
+  "MinaReceiptUC*******": [
+    "17890046087060600378523962727948914890955845280331606421027576706277370812052",
+    "2013454333407534591106411126192774509000627947643864003868625362325108814274",
+    "11843535353803037930790320689348405921362689396018494249827525941843824404081"
+  ],
```

Comment:
hard-coded hashes exposed to jsoo via the new TS bindings. we hard-code them to avoid computing them at the top level

Reply:
Nit: If we could throw some inline comments when this (or similar) file(s) is generated that point to the OCaml file, it would really help context for other developers if debugging or adding features in this area.